modular-docs-manual/content/topics/module_guidelines-procedure.adoc

```diff
@@ -30,6 +30,13 @@ The procedure consists of one or more steps required to complete the procedure.
 
 For single-step procedures, use an unnumbered bullet instead of a numbered list.
 
+[discrete]
+== Procedure Verification Steps
```

Comment:
This should be sentence case per the new guidelines for headings

Reply:
Thanks @ncbaratta . I have changed the heading to sentence case. Note that the rest of the doc is still in title case though :)